docs/content/usage-reporting.md

```diff
@@ -0,0 +1,217 @@
+- Access to public internet to pull the image, since the image is hosted in the NGINX container registry at `registry.nginx.com/cluster-connector`. Alternatively, you can pull the image down and push it to a private container registry that your cluster has access to.
+[//]: # (  TODO: Update the image and tag after published)
+
+## Setting up user accounts in NGINX Management Suite
+
+The NGINX Cluster Connector uses the NGINX Management Suite API to report the number of NGINX Ingress Controller nodes in the cluster. To use the NGINX Management Suite API, you must have a user account with the correct role.
+
+1. Create a role following the steps in [Create a Role](https://docs.nginx.com/nginx-management-suite/admin-guides/access-control/set-up-rbac/#create-role) section of the NGINX Management Suite documentation. Select these permissions in step 6 for the role:
+   - Module: Instance Manager
+   - Feature: Nginx Plus Usage
+   - Access: CRUD
+
+2. Create a user account following the steps in [Add Users](https://docs.nginx.com/nginx-management-suite/admin-guides/access-control/set-up-rbac/#add-users) section of the NGINX Management Suite documentation. In step 6, assign the user to the role created above. Note that currently only basic auth authentication is supported for usage reporting purposes.
+
+## Deploying the NGINX Cluster Connector
+
+1. Basic authentication is required to connect the NGINX Management Suite API. If you do not have a user account, create one following the above section . A base64 representation of the username and password is required to create the Kubernetes secret. In this example the username will be `foo` and the password will be `bar`. To obtain the base64 representation of a string, use the following command:
+    ```
+    > echo -n 'foo' | base64
+    Zm9v
+    > echo -n 'bar' | base64
+    YmFy
+    ```
+
+2. In order to make the credential available to NGINX Cluster Connector, we need to create a Kubernetes secret. Copying the following content to a text editor, and fill in the username and password under `data`, with base64 representation of the username and password obtained in step 1:
+    ```
+    apiVersion: v1
+    kind: Secret
+    metadata:
+      name: nms-basic-auth
+      namespace: nginx-cluster-connector
+    type: kubernetes.io/basic-auth
+    data:
+      username: Zm9v # base64 representation of 'foo' obtained in step 1
+      password: YmFy # base64 representation of 'bar' obtained in step 1
+    ```
+    Save this in a file named `nms-basic-auth.yaml`. In the example, the namespace is `nginx-cluster-connector` and the secret name is `nms-basic-auth`. The namespace is the default namespace for the NGINX Cluster Connector. If you are using a different namespace, please change the namespace in the `metadata` section of the file. Note that the cluster connector only supports basic-auth secret type in `data` format, not `stringData`, with the username and password encoded in base64.
```

Comment:
Since the cluster connector is not deployed yet, does the namespace need to be manually created?  Did we miss a step?